doc/developers/contributing.rst

```diff
@@ -489,7 +489,7 @@ In addition, we add the following guidelines:
 
 
 A good example of code that we like can be found `here
-<https://svn.enthought.com/enthought/browser/sandbox/docs/coding_standard.py>`_.
+<http://web.archive.org/web/20110527163743/https://svn.enthought.com/enthought/browser/sandbox/docs/coding_standard.py>`_.
```

Comment:
this actually (interestingly enough) has no current link. see https://github.com/scikit-learn/scikit-learn/pull/6835